jx_sqlite/aggs_table.py

```diff
@@ -542,6 +545,8 @@ def _groupby_op(self, query, frum):
         for i, s in enumerate(listwrap(query.select)):
             column_number = len(selects)
             sql_type, sql = s.value.to_sql(schema)[0].sql.items()[0]
+            if sql == 'NULL'and not s.value.var in schema.keys():
+                raise Exception('No such column' , s.value.var)
```

Comment:
Instead of 

    raise Exception('No such column' , s.value.var)

please use

    Log.error("No such column {{var}}", var=s.value.var)

The `Log` library will grab the stack trace (before it gets lost with another exception); it will build a data structure for insertion into structured logging; add a variety of annotations to the error to help trace the problem.

In general, for any project, mimic their error raising and logging style.